### PR TITLE
Customer user information center

### PR DESCRIPTION
**Pull request by nilei**
In order to not only have a detailed view on one customer, this extension allows a detailed view on a customer user. It also changes the customer information center so that a click on one of the users redirects to the customer user information center.

Help needed:
- Filters in dashboard modules don't work

### CODE REVIEW

File: Kernel/Config/Defaults.pm [hunk 1]
```diff
@@ -1364,8 +1364,8 @@ via the Preferences button after logging in.
 #        AutoLoginCreationPrefix => 'auto',
 #        # admin can change customer preferences
 #        AdminSetPreferences => 1,
-        # use customer company support (reference to company, See CustomerCompany settings)
-        CustomerCompanySupport => 1,
+#        # use customer company support (reference to company, See CustomerCompany settings)
+#        CustomerCompanySupport => 1,
```
Comment (nilei) [inline Kernel/Config/Defaults.pm]: This was changed/committed by mistake.

**Comment by nilei** [comment]
@mrcbnsls, after changing the implementation to not have so many code duplications, could you have a look into this and help me to find out why the dashboard module filters are not working?

TIA

-- Cheers, Nils

**Comment by mrcbnsls** [comment]
Hey Nils,

I have to apologize that I didn't have the opportunity to answer this PR earlier. Now I wanted to look at it and it turns out there are lots of conflicts due to the many changes in master since you submitted the PR. I'm afraid I have to ask you to update the PR and solve the conflicts before I can investigate on this any further :-(

Thanks in advance!

Marc

**Comment by mgruner** [comment]
I'm closing this for now. Let's revisit the topic in future if possible/required!

**Pull request closed without merging**